src/measure_theory/function/ae_eq_of_integral.lean

```diff
@@ -471,4 +471,52 @@ end
 
 end ae_eq_of_forall_set_integral_eq
 
+section lintegral
+
+lemma ae_eq_of_to_real_ae_eq {f g : α → ℝ≥0∞}
+  (hfi : ∀ᵐ x ∂μ, f x < ∞) (hgi : ∀ᵐ x ∂μ, g x < ∞)
```

Comment:
Could you please assume `≠ ∞`?

Reply:
Thats a very usefyl tactic, thanks!

Reply:
Do we have a lemma to convert between `∀ᵐ x ∂μ, f x < ∞` and `∀ᵐ x ∂μ, f x ≠ ∞`? I can't seem to find it